README.md

```diff
@@ -159,6 +165,219 @@ for (Bucket b : storage_service.list().iterateAll())
     System.out.println(b); 
 ```
 
+### Workload Identity Federation
+
+Using workload identity federation, your application can access Google Cloud resources from
+Amazon Web Services (AWS), Microsoft Azure or any identity provider that supports OpenID Connect
+(OIDC).
+
+Traditionally, applications running outside Google Cloud have used service account keys to access
+Google Cloud resources. Using identity federation, you can allow your workload to impersonate a
```

Comment:
" you can allow your workload to impersonate a" -->  your workload can impersonate a